spec/models/family_request_spec.rb

```diff
@@ -12,4 +12,33 @@
 require "rails_helper"
 
 RSpec.describe FamilyRequest, type: :model do
+  let(:partner) { create(:partner) }
+  let(:family_request) { create(:family_request, partner: partner) }
+
+  it { should belong_to(:partner) }
+
+  it { should have_many(:family_request_children) }
+
+  it { should have_many(:children) }
+
+  describe "a valid FamilyRequest" do
+    it "requires a Family" do
+      expect(build_stubbed(:family_request, partner: partner)).to be_valid
```

Comment:
I think there is a shoulda-matcher for validating presence that we can use in place of this. It looks like.... 
```ruby
    it { is_expected.to validate_presence_of(:associated_record) }
```